ingestion/functions/retrieval/retrieval_test.py

```diff
@@ -49,20 +49,22 @@ def test_lambda_handler_e2e(valid_event, requests_mock, s3):
     s3.create_bucket(Bucket=retrieval.OUTPUT_BUCKET)
     source_api_url = "http://foo.bar"
     origin_url = "http://bar.baz/"
+    dedupe_strategy = {"onlyParseCasesUpToDaysBefore": 2}
     os.environ["SOURCE_API_URL"] = source_api_url
     full_source_url = f"{source_api_url}/sources/{valid_event['sourceId']}"
     lambda_arn = "arn"
     requests_mock.get(
         full_source_url,
         json={"origin": {"url": origin_url}, "format": "JSON",
-              "automation": {"parser": {"awsLambdaArn": lambda_arn}}})
+              "automation": {"parser": {"awsLambdaArn": lambda_arn}},
+              "dedupeStrategy": dedupe_strategy})
```

Comment:
Seems like this was obviated by `automation.dateFilter`?

Reply:
right, forgot that I had code in there also :) fixed.